source/features/same-tab-links.tsx

```diff
@@ -0,0 +1,24 @@
+// ...existing code...
+import features from '../feature-manager.js';
+import observe from '../helpers/selector-observer.js';
+
+function removeTargetBlank(anchor: HTMLAnchorElement): void {
+	anchor.removeAttribute('target');
+}
+
+function initOnce(): void {
+	observe('a[target="_blank"]', removeTargetBlank);
+}
+
+void features.add(import.meta.url, {
+	init: initOnce,
```

Comment:
This is actually called at every page change, even without include/rzclude, so you'll need `init: onetime(init)`. You can copy the exact naming from other similar functions